website/docs/components/app-header/partials/code/how-to-use.md

```diff
@@ -44,6 +44,12 @@ probably need to be set to `true` (or omitted to rely on defaults)
 
 ### Content
 
+!!! Info
```

Comment:
(comment, non-blocker) It would be nice to figure out a way to make this work automatically for the consumers.

Reply:
I think it should be the default myself but I guess there was a reason to hold off doing so for now. Hopefully we could make that change sometime in the near future though.